components/slider.json

```diff
@@ -0,0 +1,44 @@
+{
+  "slider": {
+    "comment": "applies to sliders",
+    "figma": "https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/Components---Windows?node-id=46518%3A104257",
+    "track-empty": {
+      "#normal": {
+        "background": "@theme-control-inactive-normal"
+      },
+      "#disabled": {
+        "background": "@theme-control-inactive-disabled"
+      }
+    },
+    "track-full": {
+      "#normal": {
+        "background": "@theme-control-active-normal"
+      },
+      "#disabled": {
+        "background": "@theme-control-active-disabled"
+      }
+    },
+    "handle": {
+      "#normal": {
+        "background": "@theme-background-solid-primary-normal",
+        "border": "@theme-outline-primary-normal"
+      },
+      "#hovered": {
```

Comment:
we do not have a hover state in the component's figma, right? do we need to create one in the repo? if yes, we should we ask the momentum team to add this state to their Figma